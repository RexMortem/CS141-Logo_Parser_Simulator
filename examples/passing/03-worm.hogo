; A simple program written using forward, back, left, and right.

forward 100
right 90
forward 20
right 90
forward 100
left 90
back 50