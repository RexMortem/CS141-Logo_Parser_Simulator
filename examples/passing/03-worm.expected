[GoForward 100, TurnRight 90, GoForward 20, TurnRight 90, GoForward 100, TurnLeft 90, GoBackward 50]